2025-09-16 19:16:09,862 - ERROR - Error: HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /products (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000029D27D968E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2025-09-16 19:16:22,909 - ERROR - Error: HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /products (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000029D27DBC280>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
